docs/pages/database-access/guides/aws-cross-account.mdx

```diff
@@ -0,0 +1,196 @@
+# within AWS account `222222222222` by assuming an external AWS IAM role.
+db_service:
+  enabled: "yes"
+  aws:
+    - types: ["rds"]
+      regions: ["us-west-1"]
+      assume_role_arn: "arn:aws:iam::222222222222:role/example-role"
+      external_id: "example-external-id"
+```
+
+Restart the Teleport Database Service for the configuration file changes to take
+effect.
+
+<Admonition type="note">
+The AWS IAM role used to discover a database will also be used by the Teleport
+Database Service to provide access to that database.
+</Admonition>
+
+</TabItem>
+<TabItem label="Static Config">
+Modify your Teleport Database Service configuration file to statically register
+an AWS database in an external account and proxy connections to it.
+
+```yaml
+# This example configuration will statically register an RDS PostgreSQL instance
+# in us-west-1 within AWS account `222222222222` by assuming an external AWS
+# IAM role "example-role".
+db_service:
+  enabled: "yes"
+  databases:
+  - name: "rds-postgres"
+    protocol: "postgres"
+    uri: "rds-postgres.abcdef012345.us-west-1.rds.amazonaws.com:5432"
+    aws:
+        assume_role_arn: "arn:aws:iam::222222222222:role/example-role"
+        external_id: "example-external-id"
+```
+
+Restart the Teleport Database Service for the configuration file changes to take
+effect.
+
+</TabItem>
+<TabItem label="Dynamic Config">
+Create a dynamic database resource to dynamically register an AWS database
+in an external account and proxy connections to it.
+
+```yaml
+# This example configuration will dynamically register an RDS PostgreSQL instance
+# in us-west-1 within AWS account `222222222222`.
+# Teleport Database Service agents that match its labels with resource selectors
+# will proxy the database by assuming the configured external AWS IAM role.
+kind: db
+version: v3
+metadata:
+  name: "rds-postgres"
+  description: "Example dynamic database resource"
+  labels:
+    env: "dev"
+spec:
+  protocol: "postgres"
+  uri: "rds-postgres.abcdef012345.us-west-1.rds.amazonaws.com:5432"
+  aws:
+    # Note that account_id must match the AWS account ID in `assume_role_arn`.
+    # Dynamic database resources do not derive `account_id` from
+    # `assume_role_arn` automatically (unlike static configuration).
+    account_id: "222222222222"
+    assume_role_arn: "arn:aws:iam::222222222222:role/example-role"
+    external_id: "example-external-id"
+```
+
+Save the configuration to a file like `database.yaml` and create it with `tctl`:
+```code
+$ tctl create database.yaml
+```
+For more information about database registration using dynamic database
+resources, see: [Dynamic Registration](dynamic-registration.mdx).
+
+</TabItem>
+</Tabs>
+
+(!docs/pages/includes/aws-credentials.mdx service="the Database Service"!)
+
+## AWS IAM permissions policies
+
+AWS IAM policies must be configured for both the Teleport Database Service's
+AWS IAM identity and the external AWS IAM role:
```

Comment:
This is the first time we mention the Database Service's AWS IAM identity. I think we should mention it earlier, including any expectations we have for it. Guessing this would be an EC2 instance of Kubernetes service account?

Reply:
I mentioned it prior to this but it was at the end of the "Teleport configuration" section, which 1) doesn't quite make sense in that section and 2) is easy to miss.

So I added a section dedicated solely to that here: d937a6b496850370f5d9a9566d795ac46fcae35e

And yeah, typically customers will deploy to an EC2 instance and use an EC2 instance profile to provide it with AWS credentials.